worker/src/RTC/DtlsTransport.cpp

```diff
@@ -1241,6 +1241,7 @@ namespace RTC
 		}
 	}
 
+	// NOLINTNEXTLINE (misc-no-recursion)
```

Comment:
`SetTimeout` may call `OnTimer` which may call `SetTimeout`. I think we had this super tested and reviewed. Do you remember it @ibc?